enterprise/models.py

```diff
@@ -134,7 +136,7 @@ def logo_path(instance, filename):
     return fullname
 
 
-class EnterpriseCustomerBrandingConfiguration(TimeStampedModel):
+class EnterpriseCustomerBrandingConfiguration(TimeStampedModel):    # pylint: disable=model-missing-unicode
```

Comment:
@saleem-latif Don't suppress this - please add python_2_unicode_compatible decorator, `__unicode__`, `__str__` and tests for string conversion. You might be able to cherry-pick it from my PR where I've fixed that: https://github.com/edx/edx-enterprise/pull/8/files#diff-4bcae564bc272f5c2e6753dc6021eb1cR164.

If not, feel free to copy the methods and [tests](https://github.com/edx/edx-enterprise/pull/8/files#diff-327b130c82fa4e83e97f6c1a50a0da1cR163) - I'll resolve conflicts later